bin/install-python

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+#
+# Install each required version of Python (from the .python-version file),
+# if it's not installed already.
+#
+# Also install tox in each pyenv copy of Python, if not installed already.
+#
+# Requirements
+# ============
+#
+# * pyenv (https://github.com/pyenv/pyenv) to install versions of Python.
+#
+# Usage
+# =====
+#
+# $ ./bin/install-python
+
+# Exit if we're running on Travis CI.
+# On Travis we just want to use the versions of Python provided in the Travis
+# VM, even though they may not have be the same patch versions as in
+# .python-version.
+if [ "$TRAVIS" = "true" ]
+then
+  exit
+fi
+
+# Loop over every $python_version in the .python-version file.
+while IFS= read -r python_version
+do
+    # Install this version of Python in pyenv if it's not installed already.
+    pyenv install --skip-existing "$python_version"
+
+    # Install tox in this version of Python if it's not already installed.
+    if ! "$PYENV_ROOT/versions/$python_version/bin/tox" --version > /dev/null 2>&1
```

Comment:
On my system I ran into a problem where `PYENV_ROOT` was not defined. Perhaps it would be better to use `pyenv root` to discover that location, since the `pyenv` command must be in the PATH to get this far?